tests/message/test_tasks.py

```diff
@@ -694,3 +691,201 @@ class TaskFailureTest(
     MSG_CLASS = message.tasks.TaskFailure
     FACTORY = factories.tasks.TaskFailureFactory
     TASK_ID_PROVIDER = 'task_to_compute'
+
+#  pylint: disable=line-too-long
```

Comment:
I don't think it's justified here...

Reply:
I added it here because I wanted to create long understandable for anyone tests function names. :)

Reply:
Could this be solved by splitting this test to subclasses?
For example:

1. `TestBlenderScriptPackageResolutionValidation(BlenderScriptPackageBase)`
   1. `test_not_positive_int_list()`
   1. `test_positive_int_list()`

Reply:
@jiivan IMHO 80 characters for a line is a little bit not enough for a good test name. Can we change it for example to 120 chars per line?